charts/reporting-operator/templates/custom-resources/report-queries/pod-volume-request.yaml

```diff
@@ -165,3 +165,48 @@ spec:
     AND "timestamp" < timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}'
     GROUP BY namespace
     ORDER BY volume_request_memory_byte_seconds DESC
+
+---
+
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "cluster-persistentvolumeclaim-request"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportQueries:
+  - "persistentvolumeclaim-request-raw"
+  view:
+    disabled: true
+  columns:
+  - name: period_start
+    type: timestamp
+    unit: date
+  - name: period_end
+    type: timestamp
+    unit: date
+  - name: data_start
+    type: timestamp
+    unit: date
+  - name: data_end
+    type: timestamp
+    unit: date
+  - name: volume_request_memory_bytes_seconds
```

Comment:
we probably don't want to refer to storage as memory, since that's sort of ambiguous (could be RAM). Perhaps `volume_request_storage_bytes_seconds`?

Reply:
That's a good point. It's `spec.resources.requests.storage` in the PVC yaml. We should update the other queries too (in a separate commit).

Reply:
Actually, this is referenced everywhere as memory- including the raw query. I will open a PR for the renaming and rename the file in one separate PR